bytecomp/lambda.ml

```diff
@@ -264,9 +264,25 @@ and lambda_event_kind =
   | Lev_function
   | Lev_pseudo
 
+type module_map = {
+  map :  (int * Ident.t) list;
```

Comment:
In my patch, `map` is a list of pairs `position in gc_roots x identifier`. What would be the meaning of `symbol * int` ?
What about defining a type for the slot, i.e. 
```
type slot = Ident of Ident.t 
             | Method_cache
             | Symbol of symbol
```
This way, it would be compatible with flambda, and methcache would be included in it. What do you think ?